src/contract/ContractFunctionResult.js

```diff
@@ -57,8 +57,8 @@ export default class ContractFunctionResult {
      * @param {ContractId[]} result.createdContractIds
      * @param {Uint8Array | null} result.evmAddress
      * @param {Uint8Array} result.bytes
-     * @param {Long} result.gas
-     * @param {Long} result.amount
+     * @param {Long|null} result.gas
+     * @param {Long|null} result.amount
```

Comment:
Unfortunately, this is a breaking change. :( Would it be reasonable to perhaps use `Long.fromNumber(-1)` to represent a `null`? I would prefer using `null`, but we can't make breaking changes.